static/js/components/tiles/answer_table_tile.tsx

```diff
@@ -142,13 +165,18 @@ const fetchData = async (
     // The rest of the promises are for column values
     const propResp = resp.slice(1);
     const values = {};
+    const sources = {};
```

Comment:
It's slightly hard to follow when the same map holds provenanceIds as its values at the beginning of the function and then sourceUrls at the end. Maybe rename this to provenanceIds & then use sources later on.

Reply:
also add typing to the object